oracle/oracles/pd.go

```diff
@@ -47,19 +48,111 @@ import (
 	"github.com/tikv/client-go/v2/metrics"
 	"github.com/tikv/client-go/v2/oracle"
 	pd "github.com/tikv/pd/client"
+	"go.uber.org/atomic"
```

Comment:
Is there any reason to use the uber version rhe the original PR didn't? I think using the std one should be just fine🤔 Is this a mistake?

Reply:
std atomic was introduced in go 1.19, while release-6.5 uses go 1.18.

Reply:
😰